weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/sections/ExampleCompareSection/exampleCompareSectionUtil.ts

```diff
@@ -323,9 +302,181 @@ export const useFilteredAggregateRows = (state: EvaluationComparisonState) => {
   return useMemo(() => {
     return {
       filteredRows,
-      inputColumnKeys,
       outputColumnKeys,
       leafDims,
     };
-  }, [filteredRows, inputColumnKeys, leafDims, outputColumnKeys]);
+  }, [filteredRows, leafDims, outputColumnKeys]);
 };
+
+// Get the table digest used in the dataset of the first evaluation,
+// which prepares a request for actually fetching the table rows later
+async function getPartialTableRequest(
```

Comment:
super nit, maybe we can call this `makePartialTableReq` so that its clear this is generating a req payload.